samps/contests/interactive/config/system.pc2.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# File:    system.pc2.yaml
+# Purpose: pc2 setttings for a contest
+#
+
+
+# 8M
+max-output-size-K: 8192
+
+stop-on-first-failed-test-case: true
+team-scoreboard-display-format-string: "{:displayname}"
+
+load-data-files: false
```

Comment:
I wonder why data files are not being loaded into PC2; doing it this way requires the user/tester to deal with setting a correct Judge's Data Path, whereas if the files were loaded the user wouldn't have to do that.  The files do not seem to be very big -- a few K at most...

Reply:
I think we determined this was user error.

Reply:
We did determine that for my testing this was explained.  What I was saying was that, in general, it seems to me that our "sample contests" should not contain lines like this, which override the default setting of `true`.